udata_gouvfr/theme/templates/dataset/display.html

```diff
@@ -51,7 +51,7 @@
 
 {% block content %}
 <section class="content {% if not dataset.organization.public_service %}non{% endif %}certified">
-    <div class="container dataset-container"  itemscope="itemscope" itemtype="http://schema.org/Dataset" itemid="{{dataset.id}}">
+    <div class="container dataset-container"  itemscope="itemscope" itemtype="http://schema.org/Dataset" itemid="{{dataset.id}}" itemslug="{{dataset.slug}}">
```

Comment:
Given the schema: https://schema.org/Dataset I think you should add an extra `<meta>` tag instead:
```html
<meta itemprop="alternateName" content="{{dataset.slug}}" />
```